app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
+  # load_and_authorize_resource only: %i[edit update destroy create]
```

Comment:
- [OPTIONAL] You can add this line in `Post Controller` as you did with the `Comment Controller` to make sure to use **CanCanCan** for Post authorization

   ```
     class PostsController < ApplicationController
       load_and_authorize_resource
   
       ...
     end
   ```

Reply:
@BenMukebo , thanks a lot for the suggestion. I forgot to uncomment it.